CreeDictionary/CreeDictionary/templates/CreeDictionary/base.html

```diff
@@ -6,92 +6,112 @@
   <title>itwêwina: the online Cree dictionary</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="theme-color" content="#0091FF">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,700|Raleway&display=swap&subset=latin-ext">
+  <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,700|Raleway&display=swap&subset=latin-ext">
   <link rel="stylesheet" href="{% static 'CreeDictionary/css/styles.css' %}">
   {# magic that allows us to reverse urls in js  https://github.com/ierror/django-js-reverse    #}
   <script src="{% url 'js_reverse' %}" type="text/javascript"></script>
   <script src={% static 'CreeDictionary/js/index.js' %}></script>
+
+  {% if lemma_id and paradigm_size%}
+    {% comment %}
+    meaning we are on lemma details/paradigm page. This variable can be utilized by javascript to dynamically request
+    other things (e.g. different sized paradigms) about the lemma
+    {% endcomment %}
+    <script>
+      var lemmaId = {{ lemma_id }}
+      var paradigmSize = '{{ paradigm_size.value }}'
+    </script>
+  {% endif %}
+
 </head>
 <body class="app">
-  <a href="#start-of-content" tabindex="0" class="skip-to-content">Skip to content</a>
-  {% csrf_token %}
-  {% include './svg-sprites.html' %}
-  {% include './components/header.html' %}
+<a href="#start-of-content" tabindex="0" class="skip-to-content">Skip to content</a>
+{% csrf_token %}
+{% include './svg-sprites.html' %}
+{% include './components/header.html' %}
 
-  {% spaceless %}
-    {# The 🔊 button template. Gets inflated by JavaScript. #}
-    <template id="template:play-button">
-      <button aria-label="Play recording" class="definition__icon definition-title__play-button" data-cy="play-recording">
-        <svg class="definition-title__play-icon" focusable="false">
-          <use xlink:href="#fa-volume-up-solid" />
-        </svg>
-      </button>
-    </template>
+{% spaceless %}
+  {# The 🔊 button template. Gets inflated by JavaScript. #}
+  <template id="template:play-button">
+    <button aria-label="Play recording" class="definition__icon definition-title__play-button" data-cy="play-recording">
+      <svg class="definition-title__play-icon" focusable="false">
+        <use xlink:href="#fa-volume-up-solid"/>
+      </svg>
+    </button>
+  </template>
 
-    {# The speaker button template (multiple recordings) #}
-    <template id="template:recording-item">
-      <li class="recordings-list__item" data-cy="recordings-list__item">
-        <button class="audio-snippet"><slot name="speaker-name"></slot>, <slot name="speaker-dialect"></slot></button>
-    </template>
+  {# The speaker button template (multiple recordings) #}
+  <template id="template:recording-item">
+    <li class="recordings-list__item" data-cy="recordings-list__item">
+      <button class="audio-snippet">
+        <slot name="speaker-name"></slot>
+        ,
```

Comment:
IntelliJ is not smart about this :(

There should be a comma directly after the `<slot>` or else there will be unnecessary whitespace

```suggestion
        <slot name="speaker-name"></slot>,
```